openedx/adg/lms/applications/managers.py

```diff
@@ -67,7 +75,7 @@ def get_user_business_line_and_common_business_line_prereq_courses(self, user):
             and common courses for all the business lines
         """
         return get_courses_from_course_groups(
-            self.business_line_and_common_business_line_prereq_course_groups_for_user(user), user
+            self.business_line_and_common_business_line_prereq_course_groups(user), user
         )
```

Comment:
Please first call the `self.business_line_and_common_business_line_prereq_course_groups(user)` before the return statement, store its value in a variable, and then pass that value to the `get_courses_from_course_groups` function?

Reply:
can you please explain how will it impact? As the function is of single line so its already readable.

Reply:
Well, nested function calls usually get confusing, especially when we have such a long function name. Parameters are more readable if they contain values/variables, rather than function calls. In any case, if you feel it is readable enough, let's keep it the way it is.